cypress/integration/findings_spec.js

```diff
@@ -3,21 +3,221 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { PLUGIN_NAME } from '../support/constants';
 import {
-  PLUGIN_NAME,
-  TWENTY_SECONDS_TIMEOUT,
   TEST_INDEX,
   TEST_DETECTOR,
   TEST_DOCUMENT,
-} from '../support/constants';
+  TEST_FIELD_MAPPINGS,
+} from '../fixtures/constants';
 
 describe('Findings', () => {
   before(() => {
-    // delete any existing indices
+    // Delete any existing indices
     cy.deleteAllIndices();
+
+    // Create test index
+    cy.createIndex('cypress-test-windows', TEST_INDEX);
+
+    // Create detector manually
```

Comment:
extra comment?

Reply:
consolidated it with the one below (line 24)